docs/usage/bazel.md

```diff
@@ -5,22 +5,171 @@ description: Bazel dependencies support in Renovate
 
 # Bazel
 
-Renovate supports upgrading dependencies in Bazel `WORKSPACE` files.
+Renovate upgrades dependencies in Bazel `WORKSPACE` files and `MODULE.bazel` files.
 
 ## How it works
 
 1. Bazel support is enabled automatically
-2. Renovate will search repositories for any `WORKSPACE` files in the repository
-3. Existing dependencies will be extracted from `container_pull`, `oci_pull`, `git_repository`, `go_repository`, `maven_install`, and `http_archive`/`http_file` declarations
-4. Renovate will replace any old versions with the latest version available
+1. Renovate searches the repository for any `WORKSPACE` and `MODULE.bazel` files
+1. Renovate extracts the dependencies it finds from the files (see below for the supported dependency declarations)
+1. Renovate updates old dependencies to the latest version
 
-## git_repository
+## Bazel module (`MODULE.bazel`) support
 
-Renovate will update any `git_repository` declaration that has the following:
+### Bazel registry discovery
 
-1. name
-2. remote matching `https://github.com/<owner>/<repo>.git`
-3. tag using a valid SemVer
+Renovate searches [Bazel registries](https://bazel.build/external/registry) to find new Bazel module versions.
+You customize the registries your Bazel workspace uses by including [`--registry`](https://bazel.build/reference/command-line-reference#flag--registry) entries in your [`.bazelrc` files](https://bazel.build/run/bazelrc).
+Renovate checks the workspace's `.bazelrc` files for custom registry entries.
+If no registries are found, Renovate defaults to the [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry).
+
+Here are some important points about Renovate's Bazel registry searches.
+Renovate:
+
+- uses _all_ `--registry` flag values found in a workspace's `.bazelrc` file
+- uses any files that are transitively imported by a `.bazelrc` file
+- only uses `--registry` flag values that are not associated with [a configuration](https://bazel.build/run/bazelrc#config)
+- queries the registries in the order that they are found in the `.bazelrc` file
```

Comment:
We still mention the word `flag` here, is that intended?

Reply:
No. Good catch.